### PR TITLE
feature: Added support for Infinera XTM 

### PR DESCRIPTION
**Pull request by Torch09**
Add support for Infinera XTM Systems.

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/infinera-xtm.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+text: 'Infinera XTM'
+type: network
+icon: infinera
+mib_dir: infinera
+discovery:
+    - sysObjectID:
+        - .1.3.6.1.4.1.8708
+
+poller_modules:
+    hr-mib: false
```
Comment (Jellyfrog) [inline includes/definitions/infinera-xtm.yaml:11]: Please avoid disabling these if not necessary (bugs)

File: includes/definitions/infinera-xtm.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    ipSystemStats: false
+    netstats: false
+    ospf: false
+    ucd-mib: false
+    ports: false
+    stp: false
+discovery_modules:
```
Comment (Jellyfrog) [inline includes/definitions/infinera-xtm.yaml:18]: Same

File: tests/data/infinera-xtm_infinera-xtm-r33b-43.json [hunk 3]
```diff
@@ -0,0 +1,18317 @@
...
+    "os": {
+        "discovery": {
+            "devices": [
+                {
+                    "sysName": "<private>",
+                    "sysObjectID": ".1.3.6.1.4.1.8708",
+                    "sysDescr": "Infinera Corporation - Embedded Node Manager - ENM_CUAPPL_03A cuappl03a r33b 43",
+                    "sysContact": "<private>",
+                    "version": "r33b-43",
+                    "hardware": null,
```
Comment (Jellyfrog) [inline tests/data/infinera-xtm_infinera-xtm-r33b-43.json:11]: Isn't there any hardware info to extract?

File: includes/definitions/infinera-xtm.yaml (hunk 2 above)
Comment (Torch09) [reply includes/definitions/infinera-xtm.yaml:18]: Ok, will change that.

Only these four (hr-mib, ucd-mib, ports, stp) ?
The discovery modules are fine?

Comment (Jellyfrog) [reply includes/definitions/infinera-xtm.yaml:18]: Delete all. Including discovery. If they are not causing problems that is.

**Comment by Torch09** [comment]
Found an OID for the Hardware Name and removed the poller and discovery Modules.

**Pull request merged**